src/auth.d.ts

```diff
@@ -420,6 +420,17 @@ export namespace admin.auth {
      */
     auth_time: number;
 
+    /**
+     * The email corresponding to the user who the ID token belonged to, if available.
```

Comment:
The email of the user to whom the ID token belonged, if available.

Reply:
Done